pypdf/generic/_base.py

```diff
@@ -214,6 +214,16 @@ def __repr__(self) -> str:
         return "NullObject"
 
 
+def is_null_or_none(x: Any) -> bool:
```

Comment:
I propose adding a test for this as well to check the basic behavior.

Reply:
added